lib/pure/times.nim

```diff
@@ -258,6 +258,12 @@ elif defined(windows):
   proc localtime(a1: var CTime): ptr Tm {.importc, header: "<time.h>".}
 
 type
+  DateTimeLocale* = object
+    month_MMM*: array[12, string]
```

Comment:
I think these arrays should be indexed by their corresponding enum (`Month` and `Weekday`).

Reply:
Also, they could have more descriptive names like `monthsShort`/`monthsLong`/`weekdaysShort`/`weekdaysLong` or similar. Underscores are frowned upon in the stdlib.

Reply:
I'm fine removing underscores, but I'm not sure about `short`/`long`.  I like that `MMM` is unambiguously tied to the `MMM` pattern.  You can look it up in the documentation table.  I wonder if `short` requires a level of mental indirection. In some other localization libraries, they have even more options: short, long, standalone, narrow.

Would `monthsMMM` and `weekdaysddd` be okay?  I guess the latter one isn't very readable. hmm...

Reply:
Would just the patterns by themselves be too cryptic?

```nim
  DateTimeLocale* = object
    MMM*: array[mJan..mDec, string]
    MMMM*: array[mJan..mDec, string]
    ddd*: array[dMon..dSun, string]
    dddd*: array[dMon..dSun, string]
```